File: fix_explore.js
const fs = require('fs');
const path = 'c:/Users/Frank/OneDrive/Desktop/montapulse/pages/Explore.tsx';
let content = fs.readFileSync(path, 'utf8');
const lines = content.split('\n');

// We suspect line 926 is the extra </div>
// and 927 is the )}
// We want to remove line 926.

// Let's find the lines precisely.
const targetLine = lines[925]; // 0-indexed, so line 926 is index 925
console.log('Line 926 content:', targetLine);

if (targetLine.includes('</div>')) {
    lines.splice(925, 1);
    fs.writeFileSync(path, lines.join('\n'));
    console.log('Removed line 926');
} else {
    console.log('Line 926 did not contain </div>. Searching nearby...');
    // Fallback: search for the block
    content = content.replace(/<\/div>\s*<\/div>\s*<\/div>\s*}\)/, '</div>\n                    </div>\n                )}');
    fs.writeFileSync(path, content);
}

File: .gemini/antigravity/scratch/fix_explore.py
import sys

file_path = r'c:\Users\Frank\OneDrive\Desktop\montapulse\pages\Explore.tsx'
with open(file_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# We want to replace lines from 1096 (index 1095) to 1102 (index 1101)
# Correct sequence from 1096:
new_lines = [
    '                                            );\n',
    '                                        })}\n',
    '                                    </div>\n',
    '                                )}\n',
    '                            </div>\n',
    '                        </div>\n',
    '                    </div>\n',
    '                )}\n',
    '            </div>\n'
]

# The current lines 1095 to 1102 (1-indexed) are:
# 1095: </div>
# 1096: );
# 1097: )}
# 1098: </div>
# 1099: </div>
# 1100: </div>
# 1101: )}
# 1102: </div>

# Let's replace from line 1096 (index 1095) to 1102 (index 1101)
lines[1095:1102] = new_lines

with open(file_path, 'w', encoding='utf-8') as f:
    f.writelines(lines)
